internal/controller/common/rollout_controller.go

```diff
@@ -262,31 +261,24 @@ func GetChildName(ctx context.Context, rolloutObject RolloutObject, controller R
 
 // resolves templated definitions of a child resource with arguments for name and namespace
```

Comment:
this comment is too specific for what this function is really doing - it's really just resolving a templated definition of anything with any arguments

Reply:
Changed to be more generic.